modes/telega/evil-collection-telega.el

```diff
@@ -0,0 +1,206 @@
+;;; evil-collection-telega.el --- Evil bindings for calc -*- lexical-binding: t -*-
+
+;; Copyright (C) 2018 Pierre Neidhardt
```

Comment:
Your name

Reply:
Sorry, I forgot to modify these. Now it is done.